### PR TITLE
TASInputDlg: Minor changes

### PR DESCRIPTION
**Pull request by lioncash**
Just minor clean-up related things.

### CODE REVIEW

File: Source/Core/DolphinWX/TASInputDlg.cpp [hunk 1]
```diff
@@ -209,13 +203,15 @@ void TASInputDlg::CreateWiiLayout(int num)
   m_ext_szr->AddSpacer(space5);
   m_ext_szr->Add(nunchukaxisBox, 0, wxBOTTOM, space5);
 
-  wxGridSizer* const buttons_grid = new wxGridSizer(4);
-  for (unsigned int i = 4; i < ArraySize(m_buttons); ++i)
+  auto* const buttons_grid = new wxGridSizer(4);
+  for (size_t i = 4; i < m_buttons.size(); ++i)
```
Comment (degasus) [inline Source/Core/DolphinWX/TASInputDlg.cpp]: button : m_buttons?

File: Source/Core/DolphinWX/TASInputDlg.cpp [hunk 2]
```diff
@@ -405,25 +399,27 @@ void TASInputDlg::CreateGCLayout()
 
   const int space5 = FromDIP(5);
 
-  wxGridSizer* const buttons_grid = new wxGridSizer(4);
-  for (unsigned int i = 4; i < ArraySize(m_buttons); ++i)
+  auto* const buttons_grid = new wxGridSizer(4);
+  for (size_t i = 4; i < m_buttons.size(); ++i)
```
Comment (degasus) [inline Source/Core/DolphinWX/TASInputDlg.cpp]: auto button : m_buttons ?

**Comment by degasus** [comment]
else LGTM

File: Source/Core/DolphinWX/TASInputDlg.h [hunk 3]
```diff
@@ -33,10 +39,14 @@ class TASInputDlg : public wxDialog
...
-  static constexpr int ID_CC_R_STICK = 1004;
+  enum : int
+  {
+    ID_C_STICK = 1001,
+    ID_MAIN_STICK = 1002,
+    ID_CC_L_STICK = 1003,
+    ID_CC_R_STICK = 1004
+  };
+
   int m_eleID = 1005;
```
Comment (sepalani) [inline Source/Core/DolphinWX/TASInputDlg.h]: Does ```m_eleID``` have anything to do with the enum above? If so, shouldn't 1005 also have an entry into that enum? Is ```m_eleID``` compliant to the current coding style (or should it be addressed in another PR)?

As I'm not familiar with this, feel free to ignore this comment if I'm wrong.

File: Source/Core/DolphinWX/TASInputDlg.cpp (hunk 2 above)
Comment (lioncash) [inline Source/Core/DolphinWX/TASInputDlg.cpp]: The indexing starts at 4, not 0.

File: Source/Core/DolphinWX/TASInputDlg.h (hunk 3 above)
Comment (lioncash) [inline Source/Core/DolphinWX/TASInputDlg.h]: > Does m_eleID have anything to do with the enum above? If so...

The ones in the enum are stick IDs. The mutable one is for various other UI elements that are created on the fly depending on layout. 1005 is just the next available ID that wouldn't clash with previous ones in the event system.

>  Is m_eleID compliant to the current coding style

No, `m_ele_id` would be, though. I don't like mixing large (i.e. if I make this one conform, I'd go through and do all the others as well) stylistic changes with functional/interface changes.

>

Comment (BhaaLseN) [inline Source/Core/DolphinWX/TASInputDlg.h]: Do you think this should be documented? Just a brief comment should be enough, to make it immediately obvious that it's linked to the enum.

Comment (lioncash) [inline Source/Core/DolphinWX/TASInputDlg.h]: Yeah, that would be better. I'll add a comment about it in ~10-15 mins

File: Source/Core/DolphinWX/TASInputDlg.cpp (hunk 2 above)
Comment (riking) [inline Source/Core/DolphinWX/TASInputDlg.cpp]: Add a comment that this is skipping the d-pad buttons, then

**Pull request merged**